grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -603,7 +603,17 @@ class OccurrenceTagLib {
 
                         td(class: 'search-results-cell', *:properties) {
                             try {
-                                mkp.yieldUnescaped(formatValue(occurrence.get(column), column))
+                                def value = occurrence.get(column)
+
+                                if(column == 'scientificName') {
```

Comment:
And maybe a constant '' or "" use

Reply:
Single quotes there are on purpose. It's simple string.

Reply:
Why would anyone differentiate such a thing?

Reply:
In groovy, you can do different things with single and double quote strings.

Reply:
Double quotes can do everything single quotes can and more so there's really 0 reason to use single quotes. But oh well